en_US/security/ssl.md

```diff
@@ -66,146 +75,55 @@ File with psk identities and keys (`data/init.psk`) should be created:
 ```
 myclient1:8c701116e9127c57a99d5563709af3deaca75563e2c4dd0865701ae839fb6d79
 myclient2:d1e617d3b963757bfc21dad3fea169716c3a2f053f23decaea5cdfaabd04bfc4
-...
 ```
 
 `ssl` listener should be configured to use PSK ciphers:
 
 ```
 listeners.ssl.default {
   ...
-  ssl.versions = ["tlsv1.2"]
-  ssl.ciphers = "RSA-PSK-AES256-CBC-SHA384,RSA-PSK-AES128-GCM-SHA256,RSA-PSK-AES128-CBC-SHA256,RSA-PSK-AES256-CBC-SHA,RSA-PSK-AES128-CBC-SHA,RSA-PSK-RC4-SHA,RSA-PSK-DES-CBC3-SHA"
+  ssl_options.versions = ["tlsv1.2"]
+  ssl_options.ciphers = "RSA-PSK-AES256-CBC-SHA384,RSA-PSK-AES128-GCM-SHA256,RSA-PSK-AES128-CBC-SHA256,RSA-PSK-AES256-CBC-SHA,RSA-PSK-AES128-CBC-SHA,RSA-PSK-RC4-SHA,RSA-PSK-DES-CBC3-SHA"
   ...
 }
 
 ```
 
-## Client options
-
-SSL configuration options are common for all external resources.
-
-Example SSL configuration:
-
-```
-ssl {
-    ciphers = [
-        "TLS_AES_256_GCM_SHA384",
-        "TLS_AES_128_GCM_SHA256",
-        "TLS_CHACHA20_POLY1305_SHA256",
-        "TLS_AES_128_CCM_SHA256",
-        "TLS_AES_128_CCM_8_SHA256"
-    ]
-    depth = 10
-    enable = true
-    reuse_sessions = true
-    secure_renegotiate = true
-    verify = verify_peer
-    versions = [
-        "tlsv1.3"
-    ]
-}
-```
-
-All SSL parameters are optional. Also, for the resources that support SSL, the very `ssl` configuration section is optional too. Its absence denotes that TLS layer shouldn't be used, i.e. is equivalent to
-```
-ssl {
-    enable = false
-}
-```
-
-### `enable`
-
-Boolean field indicating whether SSL should be used for the client connection. The default is `false`.
+## SSL Client for external resources
 
-### `verify`
+Apart from allowing to receive a two-way SSL MQTT client, EMQX also provides access to external resources via SSL.
 
-Optional field, one of `verify_peer` or `verify_none`. The default value is `verify_none`. Enable(`verify_peer`) or disable(`verify_none`) peer verification.
+For example, to access a web server via HTTPS:
 
-### `versions`
+![Configure HTTPS Client](./assets/http-tls.jpg)
 
-Optional list of SSL versions to use. Possible versions are: `tlsv1.3`, `tlsv1.2`, `tlsv1.1`, and `tlsv1`.
+When creating any connection to an external resource, you can choose to enable TLS and configure its SSL certificate. As shown above:
 
-### `ciphers`
+- When **TLS Verify** is enabled, the **CA Cert** field must be filled in to verify the legitimacy of the server certificate.
+- When the server needs to verify the client certificate, then **TLS Cert** and **TLS Key** must be filled in.
+- **SNI** meanings Server Name Indication, indicates whether the server domain name and certificate are verified to be the same; A null value means no validation.
 
-Optional list of ciphers to use. The default list is:
+In addition, it is also can be configured in the `emqx.conf`:
 
 ```
-# TLS 1.3:
```

Comment:
we need these ciphers documented ?

Reply:
I think so.  Can I update it in another PR?